examples/example-gauth/src/main/java/io/grpc/examples/googleAuth/GoogleAuthClient.java

```diff
@@ -100,33 +98,31 @@ public void getTopics(String projectID) {
   }
 
   /**
-   * The app requires 2 arguments as described in
-   * @see <a href="../../../../../../GOOGLE_AUTH_EXAMPLE.md">Google Auth Example README</a>
+   * The app requires 1 argument as described in
+   * @see <a href="../../../../../../README.md">Google Auth Example README</a>
    *
-   * arg0 = location of the JSON file for the service account you created in the GCP console
-   * arg1 = project name in the form "projects/balmy-cirrus-225307" where "balmy-cirrus-225307" is
+   * arg0 = project name in the form "projects/balmy-cirrus-225307" where "balmy-cirrus-225307" is
    *        the project ID for the project you created.
    *
+   * JSON file is unnecessary on Google platforms, if it's still required somewhere else then we can set the location
+   * (path of the JSON file where its available) of JSON file to GOOGLE_APPLICATION_CREDENTIALS env variable.
    */
```

Comment:
Change to "On non-Google platforms, the GOOGLE_APPLICATION_CREDENTIALS env variable should be set to the location of the JSON file for the service account you created in the GCP console."

Reply:
Fixed